charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+## 2.22.8
+
+* Expose the `labels_as_tags` parameter of the KSM core check.
```

Comment:
Let's mention the required agent/dca versions

Reply:
Good idea !
Done: https://github.com/DataDog/helm-charts/pull/395/files#diff-0c47f15ee87d7464ff10968fa5900d272dd547e8963b2aee06a8e83537ad1bd7R6